WORKSPACE

```diff
@@ -103,3 +103,22 @@ rbe_autoconfig(
     },
     create_java_configs = False,
 )
+
+rbe_autoconfig(
+    name = "engflow_remote_config_clang_coverage",
+    digest = "sha256:239015d203837f2bdcd4cfdd710d7db60acc3fb1f002f1c926b92b42c59afdd6",
+    registry = "docker.io",
+    repository = "envoyproxy/envoy-build-ubuntu",
+    use_legacy_platform_definition = False,
+    env = {
```

Comment:
I find it surprising that you have the same env variables here and above. Are both required?